metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -282,9 +288,11 @@ def __init__(self, config: AzureADConfig, ctx: PipelineContext):
             "client_id": self.config.client_id,
             "tenant_id": self.config.tenant_id,
             "client_secret": self.config.client_secret,
-            "resource": "https://graph.microsoft.com",
-            "scope": "https://graph.microsoft.com/.default",
         }
+        if self.config.endpoint_version == "v2":
+            self.token_data["scope"] = "https://graph.microsoft.com/.default"
```

Comment:
This is potentially breaking - I've excluded `scope` by default because it doesn't seem like it is used by v1 API